src/test/java/org/sagebionetworks/bridge/S3InitializerTest.java

```diff
@@ -50,6 +51,10 @@
     @BeforeMethod
     public void beforeMethod() {
         MockitoAnnotations.initMocks(this);
+
+        // Mock config.
+        when(mockBridgeConfig.get("bucket.prop")).thenReturn(BUCKET_NAME);
+        when(mockBridgeConfig.get(S3Initializer.CONFIG_KEY_SYNAPSE_AWS_ACCOUNT_ID)).thenReturn(SYNAPSE_AWS_ACCOUNT_ID);
     }
```

Comment:
Do we verify that SYNAPSE_AWS_ACCOUNT_ID is in the template?

Reply:
Yes, on line 87.